streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -332,6 +332,15 @@ public void resume() {
             case SUSPENDED:
                 // just transit the state without any logical changes: suspended and restoring states
                 // are not actually any different for inner modules
+
+                // Deleting checkpoint file before transition to RESTORING state (KAFKA-10362)
+                try {
+                    stateMgr.deleteCheckPointFile();
+                    log.debug("Deleted check point file");
```

Comment:
nit: `log.debug("Deleted check point file upon resuming with EOS enabled");`

Reply:
I will change the log message